### PR TITLE
[15.0][MIG] report_wkhtmltopdf_param: Migration to 15.0

### PR DESCRIPTION
**Pull request by yankinmax**

### CODE REVIEW

**Comment by yankinmax** [comment]
@etobella can you pls approve the workflow?

**Comment by yankinmax** [comment]
@simahawk can you pls approve workflow?

File: report_wkhtmltopdf_param/tests/test_report_paperformat.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            with self.assertRaises(ValidationError):
+                report_paperformat.update(
+                    {"custom_params": [(0, 0, {"name": "bad-parameter"})]}
+                )
+
+    def test_wkhtmltopdf_valid_parameter(self):
+        for report_paperformat in self.env["report.paperformat"].search([]):
+            error = False
+            try:
+                report_paperformat.update(
```
Comment (simahawk) [inline report_wkhtmltopdf_param/tests/test_report_paperformat.py:23]: this could be tested w/ `assertRaises`

**Review by simahawk — APPROVED** [review]
LG

**Review by etobella — APPROVED** [review]

File: report_wkhtmltopdf_param/tests/test_report_paperformat.py (hunk 1 above)
Comment (yankinmax) [reply report_wkhtmltopdf_param/tests/test_report_paperformat.py:23]: I didn't have time to check, sorry.

**Pull request merged**